lua/autorun/client/cl_contact_forms.lua

```diff
@@ -810,4 +810,32 @@ net.Receive( "CFC_ContactForms_FailureAlert", function()
     )
 end )
 
+net.Receive("cfc_contact_forms_alert", function()
+    local data = net.ReadTable()
+    local reporter = player.GetBySteamID(data["steam_id"])
+    local reported = player.GetBySteamID(data["reported_steam_id"])
+
+    local reporterName = reporter:GetName() -- reporter's name
+    local reporterRankColor = team.GetColor(reporter:Team()) -- color of reporter's rank
+
+    local reportedName = reported:GetName() -- reported player name
+    local reportedRankColor = team.GetColor(reported:Team()) -- color of reported player rank
+
+    chat.AddText(
+        reporterRankColor, reporterName,
+        YELLOW, " has submitted a forms report on ",
+        reportedRankColor, reportedName
+    ) -- message via chat
+    LocalPlayer():PrintMessage( 4, reporterName .. " has submitted a forms report on " .. reportedName ) -- message via HUD
+    local outerColor = Color(41, 41, 41)
+    local innerColor = Color(150, 150, 150)
+    chat.AddText(
+        outerColor, "[", innerColor, "CFC Forms", outerColor, "] ",
+        Color(206,206,206), string.Left( data["message"], 99 ),
+        innerColor, "...\nFull message in console."
+    ) -- prints reporter's message
+    --PrintTable( data ) -- prints the report data in admin's console
+    print("| Reporter\t: " .. reporterName .. " (" .. data["steam_id"] .. ")\n| Reported\t: " .. reportedName .. " (" .. data["reported_steam_id"] .. ")\n| Full message\t: " .. data["message"])
```

Comment:
Maybe you could generate this from a table structure it would make it easier to add new data.
so you could have like
```
local reportData = {
    ["Reporter"] = reporterName .. " (" .. data.steam_id .. ")",
    ["Reported"] = reportedName .. " (" .. data.reported_steam_id .. ")",
    etc
}
for fieldName, fieldValue in pairs( reportData ) do
    -- print field name and value formatted the same way as there
end
```

Reply:
👌